product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfigRegistry/config_1_6_3_8.xml

```diff
@@ -0,0 +1,3 @@
+
+<manufacture>Puma</manufacture>
```

Comment:
Would <manufactur**er**> suite the context more?

Reply:
This is done.